spec/intl.html

```diff
@@ -1716,11 +1716,11 @@ <h1>
             CalendarDateFields (
               _calendar_: a String,
               _fields_: a List of Strings,
-            )
+            ): a List of Strings
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It takes a list of standard fields in _fields_ that are necessary for a given operation and returns a new list by adding relevant calendar-specific fields for the calendar represented by _calendar_. This is relevant for calendars which accept fields other than the standard set of built-in calendar fields.</dd>
+            <dd>It interprets _fields_ as the names of fields (<emu-xref href="#table-temporal-field-requirements"></emu-xref>) necessary for a given operation and returns a new List by appending the names of additional calendar-specific fields that are relevant for the built-in calendar identified by _calendar_. For example, *"era"* and *"eraYear"* are appended when _calendar_ is *"gregory"* or *"japanese"*.</dd>
```

Comment:
This should mention that the appended fields must also be relevant to the supplied fields - for example, `era` and `eraYear` are only appended when _fields_ contains `year`. Otherwise, we'd unnecessarily try to read the `era` and `eraYear` properties of PlainMonthDay in era-calendars. (In the original text, I think the word "relevant" was intended to express this, but was ambiguous.)

Reply:
Done.